### DISCUSSION TITLE
--get-O365-drive-id doesn't return for all shared libraries the url

### DISCUSSION THREAD

**Discussion by DenvE** [abraunegg/onedrive#1385] [Q&A] [answered]
Hi,

is there a way to find out why on the same onedrive, some of the shared libraries return the  drive_id and url, why some of the shared libraries don't return the drive_id?

thx

**Comment by abraunegg** [comment, marked as answer]
@DenvE 
Your discussion post is missing some critical information regarding what client version you are running.

In client version <= 2.4.10 , only the first 200 SharePoint sites are returned. PR #1329 updated how this is handled to return all sites that you have access to, not just the first 200.

 Additionally, PR #1350 added the capability to providing the 'drive_id' for multiple 'document libraries' within a single Shared Library Site.

Actions for you:
* Please check your client version, and if *not* using 'master'  (which at the moment is: `onedrive v2.4.10-24-g1203aeb`), you need to upgrade your client.
* If you are still not finding the drive_id, you need to debug what is going on using `--verbose --verbose` when added to your existing command line and open an issue ticket following the correct support process.

**Comment by DenvE** [comment]
@abraunegg 
Version of onedrive is 2.4.10.
I used the --verbose, but could not get a lot of debug info, as it should be --verbose --verbose.  Turns out, there are more then 200 sites. I will check out master, i could find back the sharepoints. One site gives  Key not found: displayName, i will look into this.

Thanks a lot for the reply.
